.github/workflows/mirror-to-ghcr.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+# This generic script aims to mirror an image from Docker hub to another registry.
+# Authentication to the registry must be done before.
+
+image=$1
+imagetag=$2
+registry=${3:-kinvolk}
+: ${tag:=latest}
```

Comment:
This line is not necessary - `tag` variable will be clobbered in the loop anyway.